src/main/java/tqs/example/impostor/repository/LockerRepository.java

```diff
@@ -0,0 +1,11 @@
+package tqs.example.impostor.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.Optional;
+
+public interface LockerRepository extends JpaRepository<Locker, Long> {
+    Optional<Locker> findById(Long id);
```

Comment:
There is no need for such method to exit. When repository extends JpaRepository a method findById is automatically declared.